interpreter/cling/lib/Interpreter/Interpreter.cpp

```diff
@@ -455,8 +455,17 @@ namespace cling {
     const char* Attr = LangOpts.CPlusPlus ? " throw () " : "";
 #else
     const char* LinkageCxx = Linkage;
+#ifdef __FreeBSD__
```

Comment:
Could we perhaps have a regular R__FREEBSD variable for consistency?

Reply:
there seems to be a R__FBSD defined in RConfig.hxx, could switch to that, but it also gets defined if __FreeBSD__ is there which also is used in a lot of places...

Reply:
Side note, there is also: `__386BSD__`, `__OpenBSD__`, `__NetBSD__`